website/docs/utilities/dialog-primitive/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,33 @@
+## Conformance rating
+
+<!-- Update conformance rating badge with correct status and remove the others -->
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
```

Comment:
I agree with @MelSumner here, this "component" is different from the others.

Reply:
Should I include any information about focus & focus order, etc. here or should I point them to information elsewhere? @MelSumner

Reply:
That's a really good question, @KristinLBradley 

WDYT about adding this to our next team meeting agenda? Maybe we should all decide it? I have a focus management doc in the work and we might just want to add a general section to our design system docs site about it. 🤔

Reply:
Thanks @MelSumner, I can include links to relevant information in the `Modal` & `Flyout` docs for now. We could look at replacing it in the future with links to the new information you are working on.

Reply:
Updated: https://hds-website-git-hds-3074-dialog-primitive-docs-hashicorp.vercel.app/utilities/dialog-primitive?tab=accessibility

How is this?

Reply:
Personally I would completely remove the "focus" sub-section at all, because it's not specific and it's not generic. We're trying to document/describe the accessibility of a primitive component, that doesn’t live as standalone entity, needs to be composed and used in contexts that we can’t know upfront.

Reply:
Ok, should I link to any kind of general information related to dialogs such as the MDN HTML dialog component docs or anything?

Reply:
I'll just remove it for now as @MelSumner mentioned she was working on more general information on focus management so perhaps we could update it later to link to that once ready.

Reply:
> Ok, should I link to any kind of general information related to dialogs such as the MDN HTML dialog component docs or anything?

@KristinLBradley good point, probably in the "code" section I would add something that explains that this component (the wrapper) is build on top of the `<dialog>` HTML element, and so this comes with some out-of-the-box functionalities in case consumers wants to leverage it (but also explain that from a styling perspective we've reset everything)